views/metabox/cpcss/generate.php

```diff
@@ -15,7 +15,7 @@
 defined( 'ABSPATH' ) || exit;
 
```

Comment:
We're checking `empty( $data['cpcss_exists'] )` multiple times in the file, let's do it once at the start of the file, then use that result after